src/client/css/variables.css

```diff
@@ -1,15 +1,8 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
 :root {
   --multiplier: 1;
   --max-width: 1200px;
   --min-width: 320px;
-  --mobile-breakpoint: 480px;
```

Comment:
This variable is being used in src/client/css/user-menu.css.

Reply:
Sorry, I think I merged this the wrong way -_- I'll fix.

Reply:
Ah, these came from me merging the MNTOR-983-settings-v2-frontend-additions branch, I'll go ahead and revert. Thanks!